packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -125,7 +127,7 @@ const config = async (env): Promise<Configuration> => ({
 
   output: {
     clean: {
-      keep: /gpx_.*/,
+      keep: new RegExp(`.*?_(amd64|arm(64)?)(.exe)?`),
```

Comment:
@jackw I implemented the change here.

I kept the `getPluginJson` method because it can be useful for future features.